storm-client/src/jvm/org/apache/storm/grouping/LoadAwareCustomStreamGrouping.java

```diff
@@ -20,5 +20,6 @@
 import java.util.List;
 
 public interface LoadAwareCustomStreamGrouping extends CustomStreamGrouping {
+   void refreshLoad(LoadMapping loadMapping);
    List<Integer> chooseTasks(int taskId, List<Object> values, LoadMapping load);
```

Comment:
OK. I was just waiting for reviewing to see if we are OK with changing the way of providing LoadMapping. Looks like you're OK with pushing so I'll remove the method.